src/google_map_markers.js

```diff
@@ -320,10 +331,36 @@ export default class GoogleMapMarkers extends Component {
           ...latLng,
         };
 
+        const markerPosition = child.props.markerPosition || 'center center';
```

Comment:
Default should be google map's default, which is bottom center

Reply:
![marker positioning](https://user-images.githubusercontent.com/38500123/126595974-e3e445fd-3468-4933-baf2-5556626f16b1.jpg)

Just for confirmation, is that the same thing referenced in the doc? 👆

Reply:
@IBanda We're changing the default position, yes, therefore we have to update the docs